app/config/services/services.xml

```diff
@@ -212,6 +212,12 @@
         <service id="AppBundle\Serializer\IdeasWorkshopSerializerNormalizer">
             <argument type="service" id="api_platform.serializer.normalizer.item" />
         </service>
+        <service id="AppBundle\Normalizer\AdherentNormalizer">
+            <argument type="service" id="api_platform.serializer.normalizer.item"/>
+            <argument type="service" id="security.token_storage" />
```

Comment:
you can omit the second argument and tag. the argument will be injected by autowiring and the tag will be added by autoconfiguring
```
<argument type="service" id="security.token_storage" />
<tag name="serializer.normalizer"/>
```

Reply:
Done.